general/src/protocol/security_report.rs

```diff
@@ -13,14 +13,18 @@ use crate::types::{Annotated, Object, Value};
 #[derive(Clone, Debug, Default, PartialEq, Empty, FromValue, ToValue, ProcessValue)]
 pub struct Csp {
     /// The directive whose enforcement caused the violation.
+    #[metastructure(pii = "true")]
```

Comment:
The fields `effective_directive`, `referrer`, and any key in `other` are not touched by the Sentry data scrubber code. Are they filtered in a different place?

Reply:
I added more pii fields than what is scrubbed in Sentry. Specifically referrer can have PII for the same reason blocked_uri and document_uri can.